sdk/test-utils/recorder-new/src/core-v2-recorder.ts

```diff
@@ -36,15 +36,16 @@ import { Matcher, setMatcher } from "./matcher";
  * This client is meant for the core-v2 SDKs(depending on core-rest-pipeline) and
  * is supposed to be passed as an argument to the recorderHttpPolicy.
  */
-export class TestProxyHttpClient {
+export class RecorderClient {
   private url = "http://localhost:5000";
   public recordingId?: string;
   public mode: string;
   private stateManager = new RecordingStateManager();
-  public httpClient: HttpClient | undefined = undefined;
+  private httpClient: HttpClient | undefined = undefined;
```

Comment:
would `private httpClient?: HttpClient;` work too? and a few below that have `| undefined`

Reply:
can can

Reply:
updated